assets/shaders/irradiance_volume_voxel_visualization.wgsl

```diff
@@ -3,8 +3,8 @@
 #import bevy_pbr::mesh_view_bindings
 
 struct VoxelVisualizationIrradianceVolumeInfo {
-    transform: mat4x4<f32>,
-    inverse_transform: mat4x4<f32>,
+    world_from_voxel: mat4x4<f32>,
+    voxel_from_world: mat4x4<f32>,
```

Comment:
Hmm. Why? ‘Voxel’ isn’t a space?

Reply:
I'm not particular about this change, but it made sense as a shorthand to me as the space in which the irradiance voxels are 1 unit in size and aligned to the origin.

Are you objecting to its name / lack of documentation, or the fact that I considered the world -> irradiance voxel matrix as a mapping between spaces?

Reply:
I’ve changed my mind after seeing the light transforms.